sdkproject/Assets/Mapbox/Unity/MeshGeneration/Modifiers/GameObjectModifiers/MapboxStylesColorModifier.cs

```diff
@@ -0,0 +1,59 @@
+﻿namespace Mapbox.Unity.MeshGeneration.Modifiers
+{
+    using Mapbox.Unity.MeshGeneration.Data;
+    using UnityEngine.Assertions;
+    using UnityEngine;
+
+    public class MapboxStylesColorModifier : GameObjectModifier
+    {
+
+        public ScriptablePalette m_scriptablePalette;
```

Comment:
Can you change variable names to follow coding style 
`Private fields should be prefixed with an underscore (_privateField) and generally avoid this keyword.`